app/api/bedrock.ts

```
@@ -0,0 +1,235 @@
+    let bodyJson;
+    try {
+      bodyJson = JSON.parse(bodyText);
+    } catch (e) {
+      throw new Error(`Invalid JSON in request body: ${e}`);
+    }
+
+    // Extract tool configuration if present
+    let tools: any[] | undefined;
+    if (bodyJson.tools) {
+      tools = bodyJson.tools;
+      delete bodyJson.tools; // Remove from main request body
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)